Automatic Validation ended with:
> 2024-04-23T11:10:27.5888598Z ##[error]Manifest Validation Failed
 2024-04-23T11:10:32.4465514Z ##[section]Finishing: Validate Manifest

(Automated response - build 877.)

--- a/manifests/b/BellSoft/LibericaJDK/17/Full/17.0.11.10/BellSoft.LibericaJDK.17.Full.installer.yaml
+++ b/manifests/b/BellSoft/LibericaJDK/17/Full/17.0.11.10/BellSoft.LibericaJDK.17.Full.installer.yaml
@@ -12,11 +12,11 @@ Installers:
   AppsAndFeaturesEntries:
   - DisplayName: Liberica JDK 17 Full (64-bit)
 - Architecture: x86
-InstallerUrl: https://download.bell-sw.com/java/17.0.11+10/bellsoft-jdk17.0.11+10-windows-i586-full.msi
-InstallerSha256: 96910F6F20045B1A602A8962D946117942C9492B299C4C123EE7A461393AC727
-ProductCode: '{C3270191-AE70-4E72-8ABB-D33852297DA4}'
-AppsAndFeaturesEntries:
-- DisplayName: Liberica JDK 17 Full (32-bit)
+  InstallerUrl: https://download.bell-sw.com/java/17.0.11+10/bellsoft-jdk17.0.11+10-windows-i586-full.msi
+  InstallerSha256: 96910F6F20045B1A602A8962D946117942C9492B299C4C123EE7A461393AC727
+  ProductCode: '{C3270191-AE70-4E72-8ABB-D33852297DA4}'
+  AppsAndFeaturesEntries:
+  - DisplayName: Liberica JDK 17 Full (32-bit)
 - Architecture: arm64
   InstallerUrl: https://download.bell-sw.com/java/17.0.11+10/bellsoft-jdk17.0.11+10-windows-aarch64-full.msi
   InstallerSha256: 1E80C5F85B8EF91196C3FB74C2A4B33FA887AAD65E74A088001BD1D94F334D22

